apps/openmw/mwmechanics/aifollow.cpp

```diff
@@ -27,9 +27,9 @@ struct AiFollowStorage : AiTemporaryBase
 
 int AiFollow::mFollowIndexCounter = 0;
 
-AiFollow::AiFollow(const std::string &actorId,float duration, float x, float y, float z)
+AiFollow::AiFollow(const std::string &actorId,float duration, float x, float y, float z, bool repeat)
 : mAlwaysFollow(false), mCommanded(false), mRemainingDuration(duration), mX(x), mY(y), mZ(z)
-, mActorRefId(actorId), mActorId(-1), mCellId(""), mActive(false), mFollowIndex(mFollowIndexCounter++)
+, mActorRefId(actorId), mActorId(-1), mCellId(""), mActive(false), mFollowIndex(mFollowIndexCounter++), mRepeat(repeat)
 {
```

Comment:
Thanks, I didn't know you could do that. I'm already in the middle of rewriting it a slightly different way, but if it's problematic I'll try the way you suggested next.